rskj-core/src/main/java/org/ethereum/net/server/Channel.java

```diff
@@ -156,17 +161,6 @@ public void initMessageCodes(List<Capability> caps) {
         messageCodec.initMessageCodes(caps);
     }
 
-    public boolean isProtocolsInitialized() {
-        return eth.isUsingNewProtocol() || eth.hasStatusPassed();
```

Comment:
- `eth.isUsingNewProtocol()` always true for RskWire
- `eth.hasStatusPassed()` always false as mentioned before